packages/cli/src/commands/index.ts

```diff
@@ -26,7 +23,6 @@ export const projectCommands = [
   upgrade,
   info,
   config,
-  doctor,
```

Comment:
This will make `doctor` not visible when run inside `react-native` project. Is this expected? I think it's a handy feature to be able to troubleshoot some issues and report the environment.

Reply:
Good point, will bring it back